tide_site.install

```diff
@@ -99,3 +101,22 @@ function tide_site_update_8004() {
   $config->set('plugin', 'entity:delete_action:node');
   $config->save();
 }
+
+/**
+ * Imports default permissions to approver and site_admin.
+ */
+function tide_site_update_8005() {
+  $roles = ['approver', 'site_admin'];
```

Comment:
@vincent-gao Tide Site does not provide those roles nor Scheduled Transitions. The permissions must be defined in Tide Core.

Reply:
ok